src/components/attendee/ChatContainer.module.scss

```diff
@@ -8,289 +8,66 @@
-          -webkit-animation: scroll-left 10s linear infinite;
-          animation: scroll-left 10s linear infinite;
-        }
-      }
-    }
-
-    #component-chat-open-button {
-      margin-left: auto;
-    }
-  }
-
-  #chat-scrollbox {
-    position: relative;
-    height: 0;
-    overflow: hidden;
-    display: flex;
-    flex-direction: column;
-    @include rounded.bottom;
-
-    @media (min-width: breakpoint.$md) {
-    }
-
-    #chat-scrollbox-mask {
-      display: flex;
-      flex-direction: column-reverse;
-      flex-grow: 1;
-      overflow: scroll;
-      mask-image: linear-gradient(
-        to top,
-        rgba(0, 0, 0, 1) 75%,
-        rgba(0, 0, 0, 0.5) 95%,
-        rgba(0, 0, 0, 0) 100%
-      );
-    }
-
-    ol {
-      display: flex;
-      flex-direction: column-reverse;
-      padding: 0 space.empty(1);
-
-      li {
-        min-height: 50px;
-        padding: space.empty(0.75) 0;
-        @include atom-chat-message;
-
-        .user-avatar {
-          width: space.button(1);
-          height: space.button(1);
-        }
-      }
-    }
-
-    form {
-      @include atom.simple-form;
-      margin: 0 0 space.empty(1);
-      background: theme.$light--canvas;
-
-      @media (prefers-color-scheme: dark) {
-        background: theme.$dark--canvas;
-      }
-
-      @media (min-width: breakpoint.$md) {
-        margin: 0 space.empty(1) space.empty(1);
-      }
     }
   }
+}
 
-  &.open {
-    #chat-scrollbox {
-      height: $-h-text-chat;
-      overflow: visible;
-    }
-
-    &.component-table-video-open {
-      #chat-scrollbox {
-        height: calc(#{$-h-text-chat} - #{layout.$h-video-chat});
-      }
-    }
-
-    @media (min-width: breakpoint.$md) {
-      height: calc(100vh - #{space.empty(2)});
-      margin-top: calc(-100vh + #{space.empty(2)} + #{space.button(1)});
-
-      &.component-table-video-open {
-        height: calc(100vh - #{space.empty(2)} - #{layout.$h-video-chat});
-        margin-top: calc(
-          -100vh + #{space.empty(2)} + #{space.button(1)} + #{layout.$h-video-chat}
-        );
-      }
-
-      display: flex;
-      flex-direction: column-reverse;
+.messageNotification {
+  background: red;
+  color: white;
+  font-size: font.$size--sm;
+  border-radius: 100%;
+  width: 6px;
+  height: 6px;
+  display: inline-block;
+  vertical-align: text-top;
+  position: absolute;
+  right: 2px;
+}
 
-      nav {
-        @include rounded.bottom;
-      }
+.toggler {
+  margin-left: auto;
+}
 
-      #chat-scrollbox {
-        flex-grow: 1;
-        @include sparkle.panel--glass;
-        @include rounded.top;
-      }
-    }
+.chatSidebar {
+  position: absolute;
+  background-color: rgba(255, 255, 255, 0.4);
+  bottom: layout.$h-chat-controls;
+  right: 0;
+  height: calc(100vh - layout.$h-chat-controls - layout.$top-nav-height);
+  @media (min-width: breakpoint.$md) {
+    top: -100vh;
+    height: calc(100vh - layout.$h-chat-controls);
+    bottom: unset;
+    width: 100%;
```

Comment:
I don't like this size logic. Maybe it belongs somewhere else. Probably in the top-level layout file.